spec/services/copy_vacancy_spec.rb

```diff
@@ -26,5 +26,28 @@
 
       Timecop.return
     end
+
+    context 'not all fields are copied' do
+      it 'should not copy fields that should be unique' do
```

Comment:
I might separate out the slug part and the counters into separate `it` blocks.  They feel like separate enough tests, with different reasons to fail.

Reply:
Separated the test into multiple individual tests, see https://github.com/DFE-Digital/teacher-vacancy-service/pull/942/commits/39703520053b37b6e1a9095598d71df36b0bfd75 for this change